src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -74,10 +75,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         if ('single_text' === $options['widget']) {
             if (self::HTML5_FORMAT === $pattern) {
-                $builder->addViewTransformer(new DateTimeToHtml5LocalDateTimeTransformer(
-                    $options['model_timezone'],
-                    $options['view_timezone']
-                ));
+                // Keep only the code inside this if in Symfony 7.0
+                if ($options['with_seconds']) {
```

Comment:
I am not really sure I understand these two branches. Effectively this means that starting with 7.0 setting `with_seconds` to `false` won't be supported anymore for HTML5 widgets, right?

Reply:
In 7.0, `with_seconds => false` won't trigger a deprecation anymore and will therefore be "allowed" and will be honored (seconds won't be displayed in the widget).

Reply:
What I am concerned about is that with the proposed solution we do not offer any upgrade path to opt-in to the new behaviour with `with_seconds` being `false` on 7.0 without getting a deprecation in 6.3/6.4.